cluster/query_executor.go

```diff
@@ -320,6 +320,16 @@ func (e *QueryExecutor) executeDropSeriesStatement(stmt *influxql.DropSeriesStat
 	return e.TSDBStore.DeleteSeries(database, stmt.Sources, stmt.Condition)
 }
 
+func (e *QueryExecutor) executeDropShardStatement(stmt *influxql.DropShardStatement) error {
+	// Remove the shard reference from the Meta Store.
+	if err := e.MetaClient.DropShard(stmt.ID); err != nil {
```

Comment:
eurgh, `e.MetaClient.DropShard(stmt.ID)` doesn't need to return an error. I took the error out to ensure we can retry safely but didn't update the signature. Will update.